pkg/cli/cli.go

```diff
@@ -0,0 +1,273 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cli
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	internalconfig "sigs.k8s.io/kubebuilder/internal/config"
```

Comment:
```suggestion
	"sigs.k8s.io/kubebuilder/internal/config"
```
I would only used the `internalconfig` alias when both `internal/config` and `pkg/model/config` are being imported in the same file.